README.md

```diff
@@ -1,19 +1,67 @@
 # project-data
+
 An experimental attempt to make a CLI for supply-chain modeling for Helpful Engineering's Project Data
 
-# Installation
+# Installation and running
 
-I'm not much of a pythonista. I've installed Anaconda to get the sympy package, and I assume any user of this will have to as well.
+Installation requires four parts:
 
-You also have to instal yamale:
+1. Clone this repository with `git@github.com:helpfulengineering/project-data-platform.git --recursive`
```

Comment:
should we use http: url instead of git url? `git clone https://github.com/helpfulengineering/project-data-platform.git`